jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -47,17 +47,42 @@
 
     <!-- Show all subregions of a subset, including Glue state and subset type. -->
     <div v-for="(region, index) in subset_definitions">
-      <v-row no-gutters>
-          <v-col>Subset type: {{ subset_types[index] }}</v-col>
+       <j-plugin-section-header style="margin: 0px; text-align: left; font-size: larger; font-weight: bold">
```

Comment:
The left-alignment and bold seem weird to me since we don't use that pattern anywhere else, but since it was requested let's leave it for now and maybe it's something we'll start using in other places.

```suggestion
       <j-plugin-section-header style="margin: 0px; margin-left: -12px; text-align: left; font-size: larger; font-weight: bold">
```